src/NeoSharp.Core/Blockchain/Genesis/IGenesisBuilder.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NeoSharp.Core.Models;
+
+namespace NeoSharp.Core.Blockchain.Genesis
+{
+    public interface IGenesisBuilder
+    {
+        Block GenesisBlock { get; }
```

Comment:
The Builder pattern assumes some method to output of the result: GenesisBlock Build(). Does it make sense?